validators/RequestSchema.js

```diff
@@ -145,6 +145,35 @@ const DeleteMediaFileRequestSchema = Joi.object().keys({
   sha: Joi.string().required(),
 })
 
+const UpdateNavigationRequestSchema = Joi.object().keys({
+  content: Joi.object()
+    .keys({
+      logo: Joi.string().allow(""),
+      links: Joi.array()
+        .items(
+          Joi.object()
+            .keys({
+              title: Joi.string().required(),
+              url: Joi.string(),
+              collection: Joi.string(),
+              resource_room: Joi.boolean().valid(true),
+              sublinks: Joi.array().items(
+                Joi.object().keys({
+                  title: Joi.string().required(),
+                  url: Joi.string().required(),
+                })
+              ),
+            })
+            .oxor("url", "collection", "resource_room")
+            .oxor("sublinks", "collection")
+            .oxor("sublinks", "resource_room")
```

Comment:
the joi docs are abit unclear on this - i'm assuming this means that as a whole, they are not required but when specified, only 1 of hte given options is allowed?